src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -333,60 +333,104 @@ There are a few ways how you can try to resolve this:
 Running End-to-End (E2E) Tests
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We use `Protractor <http://www.protractortest.org/>`__ to run our frontend E2E
-tests.
+We use `Cypress <https://www.cypress.io/>`__ to run our frontend E2E tests.
 
-Our ``run-frontend-e2e-tests.sh`` script will check if Chrome or Docker is
-installed and run the tests if either is found.
+E2E Prerequisites
+.................
+
+You need to previously build the frontend.
+
+In some environments, depending on your user permissions and the CYPRESS_CACHE_FOLDER,
+you might need to run ``npm ci`` with the ``--unsafe-perm`` flag.
+
+You might need to install adicional packages to be able to run Cypress.
+Please run ``npx cypress verify`` to verify it.
+
+run-frontend-e2e-tests.sh
+.........................
+
+Our ``run-frontend-e2e-tests.sh`` script is the go to solution when you wish to
+do a full scale e2e run.
+It will verify if everything needed is installed, start a new cluster and run
+the full suite of tests.
```

Comment:
```suggestion
the full set of tests.
```

Reply:
Both Protractor and Cypress refer to the list of tests as the `test suite`.
I will update the phrasing to match theirs.